examples/get-file-contents.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable n/no-extraneous-import */
+import {
+    PrivateKey,
+    FileContentsQuery,
+    Hbar,
+    FileCreateTransaction,
+    FileDeleteTransaction,
+    Client,
+    AccountId,
+} from "@hashgraph/sdk";
+import dotenv from "dotenv";
+
+import pino from "pino";
+import pinoPretty from "pino-pretty";
+
+dotenv.config();
+
+// Set default log level to 'silent' if SDK_LOG_LEVEL is not specified in .env
+const SDK_LOG_LEVEL = process.env.SDK_LOG_LEVEL || "SILENT";
+
+// Logger configuration based on SDK_LOG_LEVEL
+const logger = pino(
+    {
+        level: SDK_LOG_LEVEL.toUpperCase(), 
+    },
+    pinoPretty({
+        colorize: true,
+        translateTime: "SYS:standard",
+        ignore: "pid,hostname",
+    }),
+);
```

Comment:
Could you please remove this and use the **console.log()** method within the file.

Reply:
done